test/images/agnhost/BASEIMAGE

```diff
@@ -1,7 +1,7 @@
-linux/amd64=alpine:3.12
-linux/arm=arm32v6/alpine:3.12
-linux/arm64=arm64v8/alpine:3.12
-linux/ppc64le=ppc64le/alpine:3.12
-linux/s390x=s390x/alpine:3.12
+linux/amd64=alpine:3.16
```

Comment:
3.16 is the oldest version which is still receiving security fixes; I chose to move there in the interest of making the minimal changes necessary to satisfy that requirement, rather than jumping to more recent releases and potentially increasing the risk of unintended side-effects.

If the preference would be to move to 3.18, however, I'd be happy to update the PR.

Reply:
your logic sounds better than mine :)